core/src/main/scala/pureconfig/package.scala

```diff
@@ -236,9 +236,15 @@ package object pureconfig {
    * @param conf The configuration to save
    * @param outputPath Where to write the configuration
    * @param overrideOutputPath Override the path if it already exists
+   * @param options the config rendering options
    */
   @throws[IllegalArgumentException]
-  def saveConfigAsPropertyFile[Config](conf: Config, outputPath: Path, overrideOutputPath: Boolean = false)(implicit writer: Derivation[ConfigWriter[Config]]): Unit = {
+  def saveConfigAsPropertyFile[Config](
+    conf: Config,
+    outputPath: Path,
+    overrideOutputPath: Boolean = false,
+    options: ConfigRenderOptions = ConfigRenderOptions.defaults())(implicit writer: Derivation[ConfigWriter[Config]]): Unit = {
```

Comment:
This method should supply its `options` parameter to `saveConfigToStream`.

Reply:
Well spotted, fixed in 823cccb.